01-main/packages/freeplane

```diff
@@ -0,0 +1,11 @@
+DEFVER=1
+get_website "https://sourceforge.net/projects/freeplane/files/freeplane%20stable/"
+if [ "${ACTION}" != "prettylist" ]; then
+    # Find the latest .deb URL for the current naming pattern
+    local REDIR_URL="$(grep -Eo 'https://downloads\.sourceforge\.net/project/freeplane/[^"\ >]+' "${CACHE_FILE}" | grep 'freeplane_[0-9.]*~upstream-[0-9]+_all.deb' | head -n1)"
+    URL=$(unroll_url "${REDIR_URL}")
+    VERSION_PUBLISHED="$(echo "${URL}" | grep -oP 'freeplane_\K([0-9]+\.[0-9]+\.[0-9]+)(?=~upstream-1_all\.deb)')"
```

Comment:
The installed version string includes "~upstream-1" so stripping this will leave each install seeming to need an update:

```
  [+] freeplane (1.12.11~upstream-1) has an update pending. 1.12.11 is available.
```